form/2d/p5js/index.md

```diff
@@ -0,0 +1,9 @@
+---  
+layout: p5js
+author: Jeanette Knipp
+author-url: https://github.com/Lellahahaha
+title: An Obvious Circle
+tags: [P5.js, Form, Law of Closure]
+permalink: https://github.com/fabianmoronzirfas/gestalten-in-code/tree/master/form
```

Comment:
The permalink should be the category and your title e.g. 

    form/an-obvious-circle/